step4/src/app.ts

```diff
@@ -0,0 +1,21 @@
+import express, { Application, Response } from 'express';
+import path from 'path';
+import ItemsRouter from './api/items';
+import config from './config/index';
+
+const app: Application = express();
+const __dirname: string = path.resolve();
+
+app.get('/', (_, res: Response) => {
+  res.sendFile(path.join(__dirname, '/index.html'));
+});
+
+app.use(express.json());
+app.use('/api', ItemsRouter);
+app.use('/web/dist', express.static(path.join(__dirname, 'web/dist')));
```

Comment:
```suggestion
app.use(express.json());
app.use('/api', ItemsRouter);
app.use('/web/dist', express.static(path.join(__dirname, 'web/dist')));

app.get('/*', (_, res: Response) => {
  res.sendFile(path.join(__dirname, '/index.html'));
});
```
나중에 router와 spa에 대해 공부하다보면 알겠지만, 필요한 api를 모두 등록하고, 이후에 매칭되는 모든 path는 index.html을 출력하도록 해주면 좋답니다!
이 때 등록되는 순서가 중요해요~